lib/service/Service.ts

```diff
@@ -420,6 +435,13 @@ class Service extends EventEmitter {
     });
   }
 
+  private getChainConfig = (symbol: string): CurrencyConfig | undefined => {
```

Comment:
In the long run initializing a map with the symbol as key in the `constructor` might be better. What do you think?

Reply:
Yeah, we could do that but I think this would be fine for now.